src/rtc/mosaic_geobursts.py

```diff
@@ -271,26 +469,35 @@ def compute_weighted_mosaic_array(list_rtc_images, list_nlooks,
 
 
 
-def compute_weighted_mosaic_raster(list_rtc_images, list_nlooks, geo_filename,
-                                   geogrid_in=None, verbose=True):
+def mosaic_single_output_file(list_rtc_images, list_nlooks, geo_filename,
```

Comment:
What do you think about changing `geo_filename` to `mosaic_filename` for better clarity?